support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -282,7 +283,23 @@ function OneTimeCheckoutComponent({
 	);
 
 	const [otherAmountError, setOtherAmountError] = useState<string>();
-	const finalAmount = getFinalAmount(selectedPriceCard, otherAmount, minAmount);
+	const [coverTransactionCost, setCoverTransactionCost] =
+		useState<boolean>(false);
+
+	const amountWithoutCoverCost =
+		getFinalAmount(selectedPriceCard, otherAmount, minAmount, false) ?? 0;
+	const transactionCoverCost = amountWithoutCoverCost * 0.04;
+	const transactionCostCopy = `I’d like to add a further ${simpleFormatAmount(
```

Comment:
We could put the copy inside the component itself

Reply:
Yep good idea, have moved it inside.